src/starbunk/bots/strategy-loader.ts

```
@@ -20,76 +27,100 @@ interface StrategyLoaderConfig {
 		}
 
-		return isValid;
+		const requiredProps = {
+			name: 'string',
+			description: 'string',
+			processMessage: 'function'
+		} as const;
+
+		const missingProps = Object.entries(requiredProps).filter(([prop, type]) => {
+			const value = (bot as Record<string, unknown>)[prop];
+			return typeof value !== type;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)